itkwidgets/viewer.py

```diff
@@ -315,118 +456,313 @@ async def get_image(self, name: str = 'Image') -> NgffImage:
         raise ValueError(f'No image data found for {name}.')
 
     @fetch_value
-    def set_image_blend_mode(self, mode: str):
+    def set_image_blend_mode(self, mode: str) -> None:
+        """Set the volume rendering blend mode. Queue the function to be run in
+        the background thread once the plugin API is available.
+
+        :param mode: Volume blend mode. Supported modes: 'Composite',
+        'Maximum', 'Minimum', 'Average'. default: 'Composite'.
+        :type mode:  str
+        """
         self.queue_request('setImageBlendMode', mode)
     @fetch_value
-    async def get_image_blend_mode(self):
+    async def get_image_blend_mode(self) -> asyncio.Future | str:
+        """Get the current volume rendering blend mode.
+
+        :return: The future for the coroutine, to be updated with the current
+        blend mode.
+        :rtype:  asyncio.Future | str
+        """
         return await self.viewer_rpc.itk_viewer.getImageBlendMode()
 
     @fetch_value
-    def set_image_color_map(self, colorMap: str):
+    def set_image_color_map(self, colorMap: str) -> None:
```

Comment:
We should use snake_case for the arguments to keep the Python interfaces idiomatic Python when possible, i.e. colorMap -> color_map

Reply:
Yes, makes sense, thanks!